src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory_Methods.cs

```diff
@@ -1,9 +1,7 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
📝 This seems like a possible encoding mistake

Reply:
Not really sure how, but it does appear the BOM was removed. This is likely a bug in omnisharp or vscode, if I had to guess: I certainly haven't been manually mucking with BOMs.